constants/constants.ts

```diff
@@ -104,6 +104,8 @@ export const WSTETH_ADDRESS = {
 };
 
 export const LDO_ADDRESS = "0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32";
+export const BENQI_ADDRESS = "0x8729438EB15e2C8B576fCc6AeCdA6A148776C0F5";
+export const SAVAX_AVAX_TJ_POOL = "0x4b946c91C2B1a7d7C40FB3C130CdfBaf8389094d";
```

Comment:
I don't see these addresses being used, can we remove them?

Reply:
Removed.